apps/strapi-cms/config/database.ts

```diff
@@ -0,0 +1,93 @@
+import path from 'path';
+
+export default ({ env }) => {
```

Comment:
We don't need all of them, we can simplify this section. Just keep sqlite for now